runelite-client/src/main/java/net/runelite/client/plugins/customcursor/CustomCursor.java

```diff
@@ -38,7 +38,8 @@
 	DRAGON_DAGGER_POISON("Dragon Dagger (p)", "cursor-dragon-dagger-p.png"),
 	TROUT("Trout", "cursor-trout.png"),
 	DRAGON_SCIMITAR("Dragon Scimitar", "cursor-dragon-scimitar.png"),
-	CUSTOM_IMAGE("Custom Image");
+	CUSTOM_IMAGE("Custom Image"),
+	EQUIPPED_WEAPON("Equipped Weapon");
```

Comment:
`Custom Image` should be the last option imo
```suggestion
	EQUIPPED_WEAPON("Equipped Weapon"),
	CUSTOM_IMAGE("Custom Image");
```